package.json

```diff
@@ -4,13 +4,10 @@
   "description": "Agent for Kaholo",
   "main": "app.ts",
   "scripts": {
-    "build": "rimraf ./build && tsc",
+    "build": "NODE_ENV=\"production\" rimraf ./build && tsc",
```

Comment:
What for is `NODE_ENV`? In this case? Runtime env variables are not evaluated during build phase. Also can't find anything about `NODE_ENV` effect for the `tsc` itself but maybe I didn't spend enough time to find it.

Reply:
You're right when I think about it there are virtually no real scenarios to put this to use, I will remove it.